app/commands/solution/publish_iteration.rb

```diff
@@ -6,6 +6,7 @@ class Solution::PublishIteration
   def call
     solution.update!(published_iteration: iteration)
 
+    Solution::UpdateTags.(solution)
     Solution::UpdatePublishedExerciseRepresentation.(solution)
     Solution::UpdateSnippet.(solution)
     Solution::UpdateNumLoc.(solution)
```

Comment:
I'd like to check the code path for when someone publishes a new iteration on a solution that's set to "publish all iterations". Does this command get called as part of that? My guess would be that it doesn't (but I hope I'm wrong). If not, we should change it so it does call this.